"""pylint: not callable
"""

x = 10
print(x())  # Error on this line
